v0.3/cruorg/components/accordion/accordion.scss

```diff
@@ -2,3 +2,158 @@
+        svg path {
+            stroke: $cru-color-white;
+        }
+    }
+    &__button {
+        color: $cru-color-white;
+        &--expanded {
+            color: $cru-color-yellow;
+            font-weight: $cru-font-weight-bold;
+            .cmp-accordion__icon svg path {
+                stroke: $cru-color-yellow;
+            }
+        }
+    }
+    &__panel {
+        color: $cru-color-white;
+    }
+}
+
+@mixin cru-accordion-white {
+    &__button {
+        color: $cru-color-gray-dark;
+        &--expanded {
+            color: $cru-color-yellow;
+            font-weight: $cru-font-weight-bold;
+        }
+    }
+}
+
+@mixin cru-accordion-links {
+    width: 100%;
+    &__icon svg {
+        position: relative;
+        top: 10px;
+        left: 2px;
+        path {
+            stroke: $cru-color-yellow;
+        }
+    }
+    &__button {
+        background-image: linear-gradient($cru-color-yellow, $cru-color-yellow);
+        background-repeat: no-repeat;
+        background-size: 5% $cru-spacer-base / 2;
+        background-position: top left, 4px 0;
+        border: none;
+        margin: $cru-spacer-base;
+        line-height: 2rem;
+        color: $cru-color-gray-dark;
+        font-weight: $cru-font-weight-bold;
+        &--expanded {
+            .cmp-accordion__icon svg path {
+                d: path('M12 17L15.764 13.1011C15.8265 13.0364 15.9113 13 15.9997 13C16.0881 13 16.1728 13.0364 16.2353 13.1011L20 17');
+            }
+        }
+    }
+    &__panel { 
+        display: flex;
+        flex-wrap: wrap;
+        padding-left: $cru-spacer-base;
+        &--hidden {
+            display: none;
+        }
+    }
+}
+
+@mixin cru-accordion-links-black {
+    &__button {
+        color: $cru-color-white;
+    }
+}
+
+.accordion {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    padding: 5% 3%;
+    @include cru-media-breakpoint-up($cru-breakpoint-md) {
+        flex-direction: row;
+        padding: 5% 10%;
+        .cmp-accordion {
+            &__button {
+                padding-left: 0;
+            }
+        }
+    }
+    .cmp-accordion {
+        width: 100%;
+        &__header { margin: 0 }
+        &__icon {
+            svg {
+                position: absolute;
+                right: 8px;
+                top: 18px;
+            }
+        }
+        @include cru-accordion;
+    }
+    &.cru-accordion-black  .cmp-accordion {
+        @include cru-accordion-black;
+    }
+    &.cru-accordion-white .cmp-accordion {
+        @include cru-accordion-white;
+    }
+    &.cru-accordion-links {
+        .cmp-accordion {
+            @include cru-accordion-links;
+        }
+        .cru-accordion-column {
+            width: 100%;
+            @include cru-media-breakpoint-up($cru-breakpoint-md) {
+                width: 25%;
+            }
+        }
+        &.cru-accordion-black .cmp-accordion {
+            @include cru-accordion-links-black;
+        }
+    }
+}
+
```

Comment:
@sebamelzi More single-use mixins. Please put the CSS rules in its rightful place.

Reply:
@sebamelzi All three of the mixins in this file are only used once. Please move the style rules directly into the appropriate selectors.